gph/src/ripser.h

```diff
@@ -706,11 +731,18 @@ class ripser
                                         index_t n, OutputIterator out) const
     {
         --n;
-        for (index_t k = dim + 1; k > 0; --k) {
+        for (index_t k = dim + 1; k > 2; --k) {
```

Comment:
I understand why you change the condition to `k > 2` but could you just a short comment.
To shortly explains it is because the last 2 remaining vertices are computed differently just after the loop ?

Reply:
I have added these comments now.

Reply:
Perfect, I resolve the conversation then